--- XorCurses-0.1.1/Makefile~	2009-10-03 01:36:44.000000000 +0200
+++ XorCurses-0.1.1/Makefile	2009-10-07 14:55:50.000000000 +0200
@@ -3,7 +3,7 @@
 # installed in some other location or you're packaging
 # it:
 
-PREFIX=/usr/local/
+PREFIX=/usr/
 BINDIR=$(PREFIX)bin/
 SHAREDIR=$(PREFIX)share/XorCurses/
 MAPDIR=$(SHAREDIR)maps
@@ -45,7 +45,7 @@
 HEADERS:=$(wildcard *.h)
 
 $(PROG): $(OBJS)
-	$(CC) $(CFLAGS) $(OBJS) -o $(PROG) $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS) -o $(PROG) $(LIBS)
 	
 %.o : %.c $(HEADERS)
 	$(CC) $(CFLAGS) -c $<
@@ -56,10 +56,12 @@
 	rm -f $(PROG) $(OBJS)
 
 install: $(PROG)
-	install $(PROG) $(BINDIR)
-	install -d $(MAPDIR)
-	install -t $(MAPDIR) maps/[1-9]*
-	install -t $(SHAREDIR) help*.txt
+	install -d $(DESTDIR)$(BINDIR)
+	install $(PROG) $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(MAPDIR)
+	install -t $(DESTDIR)$(MAPDIR) maps/[1-9]*
+	install -d $(DESTDIR)$(SHAREDIR)
+	install -t $(DESTDIR)$(SHAREDIR) help*.txt
 
 uninstall:
 	rm -f $(BINDIR)$(PROG)
